brainstormforce/ultimate-addons-for-gutenberg
# Default design changes

### Description

- Content Timeline IconFocus color in editor and frontend.
- Carousel issue with Advanced column.

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop Content Timeline in add an IconFocus color and check it is visible in frontend as well as in the editor.
- Drag and drop Advanced column > Set child column to 50/50 > Drag and drop testimonial and post carousel and check it's working properly in Advanced column.

### Checklist:
